testing_file.R

```diff
@@ -0,0 +1,85 @@
+assessment_data_pred <- read_parquet(paths$input$assessment$local) %>%
```

Comment:
This is a testing file demonstrating that the re-converted strata are the same as the original strata.